lib/patch/file-reader.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var utils = require('../utils');
+
+function apply() {
+  utils.patchClass('FileReader');
```

Comment:
* What if the patch has been applied in lib/patch/event-target.js already ?

The event target patch patches `addEventListener` and `removeEventListener`, which `patchClass` will wrap in addition to taking care of `onXXXX` listeners / other function setters that need to be bound. The two patches achieve separate things and the order in which they're applied shouldn't matter.

* `patchClass('FileReader')` will `new FileReader()`, is that allowed ?

Yes. There's a guard in `patchClass` for environments where the class is falsey. I'm not aware of any other scenarios in which you can't new a `FileReader`.